src/core/qfieldcloudconnection.h

```diff
@@ -174,6 +180,7 @@ class QFieldCloudConnection : public QObject
     void stateChanged();
     void tokenChanged();
     void userInformationChanged();
+    void pendingAttachmentsUploaded();
```

Comment:
I guess in the future we will have a signal `pendingAttachmentUploaded(int remainingCount)` or similar. Is it possible to rename the current signal to `attachmentsUploadFinished`, so we help dyslexic people like me and make it more explicit when this signal is fired. No strong opinion on the name, but I feel like the "finished" word is a must in the name.